src/csvcubed/utils/sparql_handler/data_cube_inspector.py

```diff
@@ -207,6 +217,50 @@ def get_code_lists_and_cols(self, csv_url: str) -> CodelistsResult:
 
         return self._codelists_and_cols.get(csv_url, CodelistsResult([], 0))
 
+    def get_dataframe(self, csv_url: str) -> Tuple[pd.DataFrame, List[ValidationError]]:
+        """
+        Get the pandas dataframe for the csv url of the cube wishing to be loaded.
```

Comment:
It's be good to explain what the list of validation errors we return are about in this comment too.

Reply:
Added further comment.